test failures on ppc64

Commit 18bcc36a6996d117c156394b047ec1c0e298e202 introduced a new test and since then a build with `TARGET=POWER6` fails on ppc64 (big endian). There are follow-up fixes for Power, but they seems to fix only little-endian.
````
./openblas_utest
TEST 1/27 amax:samax [OK]
TEST 2/27 ismax:negative_step_2 [FAIL]
  ERR: test_ismin.c:88  expected 9, got 50
TEST 3/27 ismax:positive_step_2 [OK]
TEST 4/27 ismin:negative_step_2 [FAIL]
  ERR: test_ismin.c:62  expected 9, got 1
TEST 5/27 ismin:positive_step_2 [OK]
TEST 6/27 drotmg:drotmg_D1_big_D2_big_flag_zero [OK]
TEST 7/27 drotmg:rotmg_D1eqD2_X1eqX2 [OK]
TEST 8/27 drotmg:rotmg_issue1452 [OK]
TEST 9/27 drotmg:rotmg [OK]
TEST 10/27 axpy:caxpy_inc_0 [OK]
TEST 11/27 axpy:saxpy_inc_0 [OK]
TEST 12/27 axpy:zaxpy_inc_0 [OK]
TEST 13/27 axpy:daxpy_inc_0 [OK]
TEST 14/27 zdotu:zdotu_offset_1 [OK]
TEST 15/27 zdotu:zdotu_n_1 [OK]
TEST 16/27 dsdot:dsdot_n_1 [OK]
TEST 17/27 swap:cswap_inc_0 [OK]
TEST 18/27 swap:sswap_inc_0 [OK]
TEST 19/27 swap:zswap_inc_0 [OK]
TEST 20/27 swap:dswap_inc_0 [OK]
TEST 21/27 rot:csrot_inc_0 [OK]
TEST 22/27 rot:srot_inc_0 [OK]
TEST 23/27 rot:zdrot_inc_0 [OK]
TEST 24/27 rot:drot_inc_0 [OK]
TEST 25/27 potrf:smoketest_trivial [OK]
TEST 26/27 potrf:bug_695 [OK]
TEST 27/27 kernel_regress:skx_avx [OK]
RESULTS: 27 tests (25 ok, 2 failed, 0 skipped) ran in 37 ms
make[1]: *** [Makefile:45: run_test] Error 2
````

and for the record, `TARGET=POWER8` fails on ppc64 with
````
../kernel/power/icamax_power8.S: Assembler messages:
../kernel/power/icamax_power8.S:14: Error: symbol `icamax_k' is already defined
../kernel/power/icamax_power8.S:18: Error: .localentry expression for `icamax_k' does not evaluate to a constant
/tmp/ccKZt4px.s: Error: .size expression for icamax_k does not evaluate to a constant
make[1]: *** [Makefile.L1:597: icamax_k.o] Chyba 1
````
our CI runs with `TARGET=POWER6` for ppc64 (as shown above)

--- a/kernel/mips64/KERNEL
+++ b/kernel/mips64/KERNEL
@@ -167,3 +167,27 @@ endif
 
 CGEMM3MKERNEL    =  zgemm3m_kernel.S
 ZGEMM3MKERNEL    =  zgemm3m_kernel.S
+
+ifndef ISMINKERNEL
+ISMINKERNEL = imin.S
+endif
+
+ifndef IDMINKERNEL
+IDMINKERNEL = imin.S
+endif
+
+ifndef IQMINKERNEL
+IQMINKERNEL = imin.S
+endif
+
+ifndef ISMAXKERNEL
+ISMAXKERNEL = imax.S
+endif
+
+ifndef IDMAXKERNEL
+IDMAXKERNEL = imax.S
+endif
+
+ifndef IQMAXKERNEL
+IQMAXKERNEL = imax.S
+endif

--- a/kernel/power/KERNEL
+++ b/kernel/power/KERNEL
@@ -50,3 +50,26 @@ ifndef DSDOTKERNEL
 DSDOTKERNEL = ../generic/dot.c
 endif
 
+ifndef ISMINKERNEL
+ISMINKERNEL = imin.S
+endif
+
+ifndef IDMINKERNEL
+IDMINKERNEL = imin.S
+endif
+
+ifndef IQMINKERNEL
+IQMINKERNEL = imin.S
+endif
+
+ifndef ISMAXKERNEL
+ISMAXKERNEL = imax.S
+endif
+
+ifndef IDMAXKERNEL
+IDMAXKERNEL = imax.S
+endif
+
+ifndef IQMAXKERNEL
+IQMAXKERNEL = imax.S
+endif

